src/frontend/src/binder/relation/mod.rs

```diff
@@ -264,12 +274,21 @@ impl Binder {
                 } else {
                     let func_name = &name.0[0].value;
                     if let Ok(table_function_type) = TableFunctionType::from_str(func_name) {
-                        let args = args
+                        let args: Vec<ExprImpl> = args
                             .into_iter()
                             .map(|arg| self.bind_function_arg(arg))
                             .flatten_ok()
                             .try_collect()?;
-
+                        if table_function_type == TableFunctionType::RawTable {
+                            if args.len() != 1 {
+                                return Err(ErrorCode::BindError("should give only table id when using debug function __rw_table".to_string()).into());
+                            }
+                            if let ExprImpl::Literal(literal) = &args[0] {
+                                let table_id =
+                                    literal.get_data().clone().unwrap().into_int32() as u32;
```

Comment:
Will this panic on incorrect literal type? May add some plan tests for this table function with different inputs.